src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -1983,12 +1983,19 @@ def _get_principal_dics(cli_ctx, role_assignments):
     if principal_ids:
         from azure.cli.command_modules.role import graph_client_factory, GraphError
         try:
-            from azure.cli.command_modules.role.custom import _get_displayable_name, _get_object_stubs
+            from azure.cli.command_modules.role.custom import (_get_displayable_name, _get_object_stubs,
+                                                               _odata_type_to_arm_principal_type)
```

Comment:
Do we really want to import protected methods of `azure.cli.command_modules.role.custom`?

Reply:
Let me expose them in another refactoring PR.